### ISSUE TITLE
DownloadServiceFabricRuntimePackage.ps1 question

### ISSUE THREAD

**Issue by bondarewicz** [Azure-Samples/service-fabric-dotnet-standalone-cluster-configuration#2]
Hi there, I hope this would be the right place to ask a question wrt installing standalone cluster using the configs from this repo.

I was under the impression that if I download `Microsoft.Azure.ServiceFabric.WindowsServer.5.7.221.9494.zip` 
and run the following (based on [docs](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-creation-for-windows-server))
```
.\CreateServiceFabricCluster.ps1 -ClusterConfigFilePath .\ClusterConfig.Unsecure.DevCluster.json -AcceptEULA
```

then I can expect the 5.7.221 to be installed, however it appears that it downloads the latest version? from `DownloadServiceFabricRuntimePackage.ps1` 

```
#Download Runtime Package
Try
{
    $RuntimePackageDetails = Get-ServiceFabricRuntimeSupportedVersion -Latest
}
```

is there any way we can control which version of service fabric is going to be installed?

**Comment by maburlik** [comment]
Yes you can pre-download the version of your choice using the links returned by DownloadServiceFabricRuntimePackage.ps1 and when creating the cluster use parameter -FabricRuntimePackagePath.
